tests/bsim/bluetooth/host/misc/sample_test/prj.conf

```diff
@@ -0,0 +1,55 @@
+# Select only the config options that are necessary for the test.
+#
+# If the test is a stress or robustness test, it is also a good idea to set the
+# stack resources (e.g. number of buffers, roles) to the lowest value possible.
+
+CONFIG_BT=y
+CONFIG_BT_DEVICE_NAME="sample-test"
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
+
+# Dependency of testlib/adv and testlib/scan.
+CONFIG_BT_EXT_ADV=y
+
+CONFIG_BT_GATT_CLIENT=y
+CONFIG_BT_GATT_AUTO_DISCOVER_CCC=y
+
+# This is the object of the test. Commenting it out should make the test fail.
+CONFIG_BT_GATT_ENFORCE_SUBSCRIPTION=n
+
+# Always enable asserts in tests, they're virtually free on native.
+CONFIG_ASSERT=y
```

Comment:
If we suggest to always enable this, then maybe it should be enabled by default

Reply:
probably (also for other comment).
I will make an issue/PR for this, because people gon complain.